src/oas3/guards.ts

```diff
@@ -1,7 +1,20 @@
 import { Dictionary } from '@stoplight/types/dist';
 import { isObject } from 'lodash';
-import { SecuritySchemeObject } from 'openapi3-ts';
+import { HeaderObject, SecuritySchemeObject } from 'openapi3-ts';
+import { BaseParameterObject } from 'openapi3-ts/src/model/OpenApi';
```

Comment:
Don't like importing from `src` directory, and seems unimportant in usage below?

Reply:
Ooops, good catch. WebStorm does it quite often when I perform automatic import insertion.